src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -657,7 +658,8 @@ private Conversion ClassifyStandardImplicitConversion(BoundExpression sourceExpr
 
             Conversion conversion = ClassifyImplicitBuiltInConversionFromExpression(sourceExpression, source, destination, ref useSiteInfo);
             if (conversion.Exists &&
-                !conversion.IsInterpolatedStringHandler)
+                !conversion.IsInterpolatedStringHandler &&
+                !conversion.IsCollectionLiteral)
```

Comment:
Comment above needs to be updated as well, last paragraph calling out exceptions.